dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/domain/DataValue.java

```diff
@@ -65,8 +65,8 @@
     private String value;
 
     @JsonProperty
-    private String createdBy;
+    private UserInfo createdBy;
 
     @JsonProperty
-    private String lastUpdatedBy;
+    private UserInfo lastUpdatedBy;
```

Comment:
should/could this be updatedBy? like the Enrollment.updatedBy, ...

Reply:
I agree.
Maybe it is good to have a 👍 from @Bekkalizer or @stian-sandvold

Reply:
Let's do updatedBy, I think we had a discussion about this way back, and decided to rename these data and user fields to created/updated + "..At" and "...By"

Reply:
The doubt here was about lastUpdatedBy or updatedBy.
In NTI we are using updatedBy, so I changed all lastUpdatedBy fields

Reply:
Yeah, lets go to updatedBy :)